buildpack.toml

```diff
@@ -19,27 +19,29 @@ api = "0.7"
     composer = "*"
 
   [[metadata.dependencies]]
-    cpe = "cpe:2.3:a:getcomposer:composer:2.4.3:*:*:*:*:*:*:*"
+    checksum = "sha256:26d72f2790502bc9b22209e1cec1e0e43d33b368606ad227d327cccb388b609a"
+    cpe = "cpe:2.3:a:getcomposer:composer:2.4.3:*:*:*:*:python:*:*"
     id = "composer"
+    licenses = ["MIT"]
     name = "composer"
     purl = "pkg:generic/composer@2.4.3?checksum=26d72f2790502bc9b22209e1cec1e0e43d33b368606ad227d327cccb388b609a&download_url=https://getcomposer.org/download/2.4.3/composer.phar"
-    sha256 = "26d72f2790502bc9b22209e1cec1e0e43d33b368606ad227d327cccb388b609a"
     source = "https://getcomposer.org/download/2.4.3/composer.phar"
-    source_sha256 = "26d72f2790502bc9b22209e1cec1e0e43d33b368606ad227d327cccb388b609a"
-    stacks = ["io.buildpacks.stacks.bionic"]
-    uri = "https://deps.paketo.io/composer/composer_2.4.3_linux_noarch_any-stack_26d72f27.phar"
+    source-checksum = "sha256:26d72f2790502bc9b22209e1cec1e0e43d33b368606ad227d327cccb388b609a"
+    stacks = ["*"]
+    uri = "https://getcomposer.org/download/2.4.3/composer.phar"
     version = "2.4.3"
 
   [[metadata.dependencies]]
-    cpe = "cpe:2.3:a:getcomposer:composer:2.4.4:*:*:*:*:*:*:*"
+    checksum = "sha256:c252c2a2219956f88089ffc242b42c8cb9300a368fd3890d63940e4fc9652345"
+    cpe = "cpe:2.3:a:getcomposer:composer:2.4.4:*:*:*:*:python:*:*"
```

Comment:
For posterity, should the `python` bit be here? The NIST database returns no matches either way so not sure which is correct.

Reply:
@paketo-buildpacks/php-maintainers

Reply:
@thitch97 Sorry I think I merged this just before your comment